<launch>
  <arg name="usb_port"                default="/dev/ttyUSB0"/>
  <arg name="dxl_baud_rate"           default="1000000"/>

  <node name="yaren_motor_data" pkg="yaren_u2d2" type="w_datos.py" output="screen">
    <param name="usb_port" value="$(arg usb_port)" />
    <param name="dxl_baud_rate" value="$(arg dxl_baud_rate)" />
  </node>

</launch>
